src/pate-chinois.md

```diff
@@ -0,0 +1,60 @@
+# Pâté Chinois (Quebec Style Shepherd’s Pie)
+
+This is the most traditional dish that every household eats in Quebec. As a kid, you would eat Pâté Chinois at least once a week, either at home or at the school's cafeteria,. It is very similar to a Shepherd’s Pie, as it is divided in 3 layers of ingredients: meat, legumes and mashed potatoes. The literal translation of the name is something like _chinese pie_. You can search for the origin of the name on the internet, I'm not an history teacher.
+
+
+- ⏲️ Prep time: 20 min
+- 🍳Cook time: 40-60 min
+- 🍽️ Servings: 6-8
+
+## Ingredients
+
+### Mashed potatoes layer
+* 1½ lb of potatoes or around 6 big one, peeled and diced
+* 2 tbsp of butter
+* ½ cup milk
+* 1 tsp of  herbs de Provence or dried parsley
+
+* (optional) ½ cup of strong cheddar cheese, grated 
+* Pinch of nutmeg
+* Paprika
+
+### Beef layer
+* 1 tbsp of butter
+* 1 lb of ground beef
+* 1 onion, diced
+* 1 tbsp of soya sauce
+* 1 tbsp of herbs de Provence
+
+* 1 tsp of Worcestershire sauce
+
+
+### Corn layer
```

Comment:
```suggestion
* 1 tbsp of herbs de Provence
* 1 tsp of Worcestershire sauce

### Corn layer
```

Please remove extra empty line, and the empty line between 2 ingredients

Reply:
Extra line removed